assembly/utils/Bytes.ts

```diff
@@ -25,13 +25,80 @@ export class Bytes {
         return result;
     }
 
-    static copyFromPosition (input: u8[], out: Array<u8>, position: i32 = 0): void {
-        for (let i = 0; i < out.length; i++) {
-            if (input.length <= i) {
+    static copyFromPosition (src: u8[], dst: Array<u8>, position: i32 = 0): void {
+        for (let i = 0; i < dst.length; i++) {
+            if (src.length <= i) {
                 break;
             }
-            out[position + i] = input[i];
+            dst[i] = src[position + i];
         }
     }
+
+    static copyToPosition (src: u8[], dst: Array<u8>, position: i32 = 0): void {
+        for (let i = 0; i < dst.length; i++) {
+            if (src.length <= i) {
+                break;
+            }
+            dst[position + i] = src[i];
+        }
+    }
+
+    static decodeInt (input: u8[]): i64 {
+        const result = Bytes.decodeUint(input);
+        return i64(result);
+    }
+
+    static decodeUint (input: u8[]): u64 {
+        const mode = input[0] & 3;
```

Comment:
I think that we need to handle the use-cases in which there is no element at position `0`. Maybe we need to do a check beforehand?